core-services/src/main/java/org/zalando/nakadi/service/publishing/NakadiKpiPublisher.java

```diff
@@ -38,7 +53,46 @@ public void publish(final String etName, final Supplier<JSONObject> eventSupplie
                 return;
             }
 
-            eventsProcessor.queueEvent(etName, eventMetadata.addTo(eventSupplier.get()));
+            jsonEventsProcessor.queueEvent(etName, eventMetadata.addTo(eventSupplier.get()));
+        } catch (final Exception e) {
+            LOG.error("Error occurred when submitting KPI event for publishing", e);
+        }
+    }
+
+    public void publish(final String eventTypeName,
```

Comment:
This looks very specific to publishing the _access log_ events, so I suggest we call it so and remove `eventTypeName` parameter:
```suggestion
    public void publishAccessLogEvent(
```